.vscode/tasks.json

```diff
@@ -292,6 +316,35 @@
                 "telink-tlsr9518adk80d-light",
                 "tizen-arm-light"
             ]
+        },
+        {
+            "type": "pickString",
+            "id": "efr32App",
+            "description": "What mbed application do you want to use?",
+            "options": [
+                "light",
+                "light-rpc",
+                "lock",
+                "unit-test",
+                "window-covering"
+            ],
+            "default": "light"
+        },
+        {
+            "type": "pickString",
+            "id": "efr32Board",
+            "description": "What EFR32 board are you building for?",
+            "options": [
+                "brd4161a",
```

Comment:
add these to exampleTarget ? That list is autocomplete so just having  it super large does not seem very painful.

Reply:
Ok, i brought extended exampleTarget to contain all examples and boards and removed the new pickStrings I previously added